src/components/BrowseProgramCard.tsx

```diff
@@ -0,0 +1,104 @@
+            <WrapItem flexBasis="300px" flexGrow={1}>
+                <Box
+                    borderWidth="1px"
+                    width="100%"
+                    _hover={{
+                        borderColor: colourTheme.colors.Blue,
+                        borderWidth: 1,
+                    }}
+                >
+                    <Box minWidth="340px" height="252px" p="35px">
+                        <HStack>
+                            <Link
+                                params={{ cardInfo: cardInfo }}
+                                href={`/admin/classes?programId=${cardInfo.id}`}
+                            >
+                                <Box
+                                    mt="1"
+                                    fontWeight="600"
+                                    fontSize="18px"
+                                    width="218px"
+                                >
+                                    {cardInfo.name}
+                                </Box>
+                            </Link>
+                            <Spacer />
+                            <Button padding="2px" borderRadius="full">
+                                <IoEllipsisVertical />
+                            </Button>
+                        </HStack>
+                        <Box as="span" color="gray.600" fontSize="sm">
+                            {t("time.range", {
+                                ...convertToShortDateRange(
+                                    cardInfo.startDate,
+                                    cardInfo.endDate,
+                                    router.locale as locale,
+                                ),
+                            })}
+                        </Box>
+                        <Box
+                            whiteSpace="pre-wrap"
+                            noOfLines={2}
+                            overflow="hidden"
+                            width="275px"
+                            height={9}
+                            mt="2"
+                            fontSize="12px"
+                        >
+                            {cardInfo.description}
+                        </Box>
+                        <Box mt={6}>
+                            <SDCBadge
+                                backgroundColor={
+                                    colourTheme.colors.CatskillWhite
+                                }
+                                color={colourTheme.colors.Blue}
+                                children={cardInfo.tag}
+                            />
+                            <SDCBadge
+                                ml="2"
+                                backgroundColor={
+                                    colourTheme.colors.CatskillWhite
+                                }
+                                color={colourTheme.colors.Blue}
+                                children={cardInfo.onlineFormat}
+                            />
```

Comment:
use existing component:
```suggestion
                            <AdminBadge>{cardInfo.tag}</AdminBadge>
                            <AdminBadge ml={2}>
                                {cardInfo.onlineFormat}
                            </AdminBadge>
```